src/test/pybind/test_rbd.py

```diff
@@ -2882,6 +2882,38 @@ def test_group_image_remove(self):
         with Image(ioctx, image_name) as image:
             eq(0, image.op_features() & RBD_OPERATION_FEATURE_GROUP)
 
+    def test_group_snap_get_info(self):
+        self.image_names.append(create_image())
+        self.image_names.sort()
+        self.group.add_image(ioctx, self.image_names[0])
+        self.group.add_image(ioctx, self.image_names[1])
+        pool_id = ioctx.get_pool_id()
+
+        self.group.create_snap(snap_name)
+
+        snap_info_dict = self.group.get_snap_info(snap_name)
```

Comment:
> Missing a negative test for a non-existing snapshot, expecting an exception.

... and one passing an empty string could be added here, where there is some existing snapshot.